testsuite/tests/libratest/smoke_test.rs

```diff
@@ -60,8 +60,13 @@ impl TestEnvironment {
         let mut template = NodeConfig::default_for_validator();
         template.state_sync.chunk_limit = chunk_limit;
 
-        let validator_swarm =
-            LibraSwarm::configure_validator_swarm(num_validators, None, Some(template)).unwrap();
+        let validator_swarm = LibraSwarm::configure_validator_swarm(
+            &workspace_builder::get_libra_node_with_inject_error(),
```

Comment:
seems I misread something, so we generate the builder here way before generating the configs

Reply:
Yeah so before `prepare` was called first thing when we enter `configure_validaotr_swarm`, now we're forcing users to provide us with a path to the binary so `prepare` becomes redundant because the binaries are already all built.